connectors/kafka/src/test/java/io/camunda/connector/kafka/inbound/KafkaExecutableTest.java

```diff
@@ -172,11 +172,11 @@ void testActivateAndDeactivate_consumerThrows() {
         .atMost(Duration.ofSeconds(5))
         .pollInterval(Duration.ofMillis(500))
         .untilAsserted(() -> assertFalse(kafkaExecutable.kafkaConnectorConsumer.shouldLoop));
-    kafkaExecutable.deactivate();
```

Comment:
This was removed because it's not really needed and the test was flaky because of it (sometimes it was deactivated before checking, sometimes not, resulting in a varying number of close() calls)